resources/views/ama_announcements/create.blade.php

```diff
@@ -0,0 +1,31 @@
+@extends('layouts.app')
+@section('content')
+
+<h1>Create Announcement</h1>
+
+<!-- if there are creation errors, they will show here -->
+{{ Html::ul($errors->all()) }}
+
+{{ Form::open(array('url' => 'ama_announcements')) }}
+
+    <div class="form-group">
+        {{ Form::label('title', 'Title') }}
+        {{ Form::text('title', old('title'), array('class' => 'form-control')) }}
+    </div>
+
+    <div class="form-group">
+        {{ Form::label('text', 'Text') }}
+        {{ Form::text('text', old('text'), array('class' => 'form-control')) }}
+    </div>
+
+    <div class="form-group">
```

Comment:
Remove this. The user id is a foreign key and should be taken from the authenticated user.